opmon/android-add-ons-extension-process.toml

```diff
@@ -0,0 +1,25 @@
+[project]
+
+name = "Android Add Ons Extension Process"
+platform = "fenix"
+metrics = [
+  "storage_stats_app_bytes",
+  "storage_stats_cache_bytes",
+  "storage_stats_data_dir_bytes",
+  "performance_page_non_blank_paint"
+]
+
+[project.population]
+
+data_source = "baseline"
+channel = "nightly"
+
+
+[metrics]
+
+[metrics.performance_page_non_blank_paint]
+data_source = "metrics"
+select_expression = "ARRAY_AGG(metrics.timing_distribution.performance_page_non_blank_paint IGNORE NULLS)"
+friendly_name = "Page Non Blank Paint"
+description = "The time between navigationStart and the first non-blank paint of a foreground root content document, in milliseconds."
+type = "histogram"
```

Comment:
Adding `performance_page_non_blank_paint` here for now. We can make it available across all configs as well if this is something that will be used in other rollouts as well

Reply:
It will be used across rollouts we're doing.